Source/Inno-Setup-Script/Mosa-Installer.iss

```diff
@@ -50,52 +48,15 @@ Source: "..\..\Tools\readme.md"; DestDir: "{app}\Tools"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Platform.x86.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Platform.x64.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Tool.Explorer.exe"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Tool.Debugger.exe"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Tool.Launcher.exe"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Tool.Compiler.exe"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Tool.Launcher.Console.exe"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Utility.BootImage.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Utility.Configuration.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Utility.DebugEngine.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Utility.FileSystem.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Utility.Launcher.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Utility.RSP.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Mosa.Utility.Disassembler.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-;Source: "..\..\bin\System.Threading.Channels.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-;Source: "..\..\bin\System.Threading.Tasks.Extensions.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\dnlib.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\CommandLine.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\PriorityQueue.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\WinFormsUI.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\WeifenLuo.WinFormsUI.Docking.ThemeVS2015.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Reko.Arch.Arm.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Reko.Arch.X86.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Reko.Core.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\bin\Reko.dll"; DestDir: "{app}\bin"; Flags: ignoreversion
-Source: "..\..\Tools\qemu\*.*"; DestDir: "{app}\Tools\qemu"; Flags: ignoreversion
-Source: "..\..\Tools\wget\*.*"; DestDir: "{app}\Tools\wget"; Flags: ignoreversion
-Source: "..\..\Tools\nasm\*.*"; DestDir: "{app}\Tools\nasm"; Flags: ignoreversion
-Source: "..\..\Tools\ndisasm\*.*"; DestDir: "{app}\Tools\ndisasm"; Flags: ignoreversion
-Source: "..\..\Tools\Bochs\*.*"; DestDir: "{app}\Tools\Bochs"; Flags: ignoreversion
-Source: "..\..\Tools\7zip\*.*"; DestDir: "{app}\Tools\7zip"; Flags: ignoreversion
-Source: "..\..\Tools\mkisofs\*.*"; DestDir: "{app}\Tools\mkisofs"; Flags: ignoreversion
-;Source: "..\..\Tools\rufus\*.*"; DestDir: "{app}\Tools\rufus"; Flags: ignoreversion
+Source: "..\..\bin\*.*"; DestDir: "{app}\bin"; Flags: ignoreversion recursesubdirs
```

Comment:
Includes too much

Reply:
It is a better solution though, because manually adding files will take too much time.

Reply:
It was working before (so not too much time).

Reply:
It didn't add some required DLLs, so it wouldn't actually work.

Reply:
Which ones?

Reply:
Nevermind, that was by hardcoding the paths I think.